src/components/Page/README.md

```diff
@@ -391,5 +391,7 @@ Title metadata appears immediately after the page’s title. Use it to communica
 
 ## Related components
 
-- To lay out the content within a page, [use the layout component](/components/structure/layout)
-- When using `Page` within an [embedded app](https://github.com/Shopify/polaris-react/blob/master/documentation/Embedded%20apps.md), rendering is delegated to the Shopify App Bridge
+- To lay out the content within a page, use the [layout component](/components/structure/layout)
+- To add pagination to a page, see the [pagination component](/components/navigation/pagination)
+- To add primary and secondary calls to action at the bottom of a page, see the [page actions component](/components/structure/page-actions)
+- When you use the page component within an [embedded app](https://github.com/Shopify/polaris-react/blob/master/documentation/Embedded%20apps.md), rendering is delegated to the Shopify App Bridge
```

Comment:
Sometimes we do `Page` to indicate that is referencing a react component, I think your change to `page` is fine in this context.